presto-geospatial/src/main/java/com/facebook/presto/geospatial/BingTileOperators.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.geospatial;
+
+import com.facebook.presto.spi.function.ScalarOperator;
+import com.facebook.presto.spi.function.SqlType;
+import com.facebook.presto.spi.type.AbstractLongType;
+import com.facebook.presto.spi.type.StandardTypes;
+
+import static com.facebook.presto.spi.function.OperatorType.EQUAL;
+import static com.facebook.presto.spi.function.OperatorType.HASH_CODE;
+import static com.facebook.presto.spi.function.OperatorType.NOT_EQUAL;
+
+public final class BingTileOperators
```

Comment:
do we have any tests to test these operators?

Reply:
@nezihyigitbasi , I'm adding tests for equal and notEqual operators, but I don't know how to test hash_code operator. Any suggestions?

Reply:
One way is to write a query to join on a key of type `BingTileType`, but I don't see any integration tests for the `GeoPlugin`. So, I guess for now we can skip this unless other reviewers think we should add it.